src/components/Navigations/NavItem/NavItem.js

```diff
@@ -0,0 +1,15 @@
+import React, {Component} from 'react';
+import classes from './NavItem.module.css';
+import {NavLink} from "react-router-dom";
+
+class NavItem extends Component {
+    render() {
+        return (
+            <li className={classes.NavItem}>
+                <NavLink exact activeClassName={classes.active} to={this.props.link} >{this.props.children}</NavLink>
```

Comment:
It is usually preferred to do props destructuring: https://dev.to/arnas/destructuring-react-props-for-the-cleaner-code-293